web/styles/left_sidebar.css

```diff
@@ -475,24 +485,41 @@ ul.filters {
         }
     }
 
-    & li.muted_topic,
-    li.out_of_home_view {
-        color: hsl(0deg 0% 20% / 50%);
-
-        .has-unmuted-mentions .unread_mention_info {
-            color: hsl(0deg 0% 20%);
+    & li.muted_topic {
+        .sidebar-topic-check,
+        .sidebar-topic-name,
+        .unread_count {
+            opacity: var(--opacity-left-sidebar-muted);
         }
 
-        .stream-privacy {
-            opacity: 0.5;
+        &:hover {
+            .sidebar-topic-check,
+            .sidebar-topic-name,
+            .unread_count {
+                opacity: var(--opacity-left-sidebar-muted-hover);
+            }
         }
+    }
 
-        & .unread_count {
-            opacity: 0.5;
+    & li.out_of_home_view {
+        .stream-privacy,
+        .stream-name,
+        .channel-new-topic-button,
+        .unread_count,
+        .masked_unread_counter,
```

Comment:
```
left_sidebar: Simplify consistent opacity for muted channels, topics.
```

`.masked_unread_count`, I think? (Here and below.)

Reply:
Oh, thank you for catching this! Fixed in the latest push.